examples/tictactoe/src/app.tsx

```diff
@@ -16,10 +22,38 @@ export default function TicTacToe() {
 
   return (
     <div className={"outerDiv"}>
-      <Board squares={board} onClick={(i) => handleClick(i)} />
-      <div className="game-info">
-        <div>{status}</div>
-        <HistoryButtons history={history} onClick={jumpTo} />
+      <div className={"headerDiv"}>
+        <img src="../public/logo.png" alt="logo" width="130" height="50" />
+      </div>
+      <div className={"titleDiv"}>
+        <h1 style={{ fontSize: "4rem" }}>Verifiable Tic Tac Toe</h1>
+      </div>
+      <div className={"gameDiv"}>
+        <Board
+          enableClick={isInitialized}
+          squares={board}
+          onClick={(i) => handleClick(i)}
+        />
+        <GameInfo
+          isInitialized={isInitialized}
+          assertionQueue={assertionQueue}
+          status={status}
+          isGameFinished={finished}
+          onClickReplay={() => {
+            replayGame();
+          }}
+        />
+      </div>
+      <div className={"footerDiv"}>
+        <p> Made with ❤️ by Kryha</p>
```

Comment:
Make all the footer content be in one line. 
Both links can be in the same `<p>`.
Also remember to add the link to Kryha's homepage to the word "Kryha".